acceptance_tests/features/steps/display_respondent_alongside_response_status.py

```diff
@@ -0,0 +1,122 @@
+def survey_is_set_to_completed_by_telephone(context):
+    signed_in_respondent(context)
+    go_to_history_tab()
+    assert browser.find_by_text('Completed by phone')
+
+
+@given('the status is set to "No longer required"')
+def survey_set_to_longer_required(context):
+    signed_in_respondent(context)
+    go_to_history_tab()
+    assert browser.find_by_text('No longer required')
+
+
+@given('an internal user has set a given survey to "No longer required" for a given respondent')
+def survey_is_set_to_no_longer_required(context):
+    signed_in_internal(())
+    go_to_reporting_unit_page(context)
+    reporting_unit.click_data_panel(context.short_name)
+    reporting_unit.click_change_response_status_link(survey=context.short_name, period=context.period)
+    mapped_status = map_ce_status('No longer required')
+    change_response_status.update_response_status(mapped_status)
+
+
+@given('a user has partially completed a survey (In progress)')
+def respondent_partially_completes_survey(context):
+    signed_in_respondent(context)
+    browser.click_link_by_id('access_survey_button_1')
+    browser.click_link_by_id('download_survey_button')
+    time.sleep(5)
+
+
+@given('the status is set to "In progress"')
+def survey_set_to_in_progress(_):
+    surveys_todo.go_to()
+    assert browser.find_by_text('Downloaded')
+
+
+@given('has not initiated any changes to the "Not started" status')
+def no_changes_have_been_made_to_collex(_):
+    assert browser.find_by_text('Not started')
+
+
+@given('the status is set to Completed')
+def status_set_to_completed(_):
+    surveys_todo.go_to()
+    go_to_history_tab()
+    assert browser.find_by_text('Complete')
+
+
+@when('an internal user navigates to Reporting units for that Reference')
+def navigating_to_reporting_units(context):
+    signed_in_internal(())
+    go_to_reporting_unit_page(context)
+    reporting_unit.click_data_panel(context.short_name)
+    reporting_unit.click_change_response_status_link(survey=context.short_name, period=context.period)
+
+
+@when('an internal user navigates to Reporting Units for the no longer required reference')
+def collex_set_to_no_longer_required(context):
+    signed_in_internal(())
+    go_to_reporting_unit_page(context)
+    reporting_unit.click_data_panel(context.short_name)
+
+
+@then('no longer required should be displayed in the survey collection exercise')
+def no_longer_required_set_for_collex(_):
+    assert 'No longer required' in browser.html
+
+
+@then('no Respondent name should be displayed in the Respondent field')
+def respondent_name_is_not_displayed(_):
+    assert 'Respondent:' in browser.html
+    assert browser.find_by_id('completed-respondent')[0].value is ''
+
+
+@then('the Respondent details must be displayed in the Respondent field')
+def respondent_name_is_displayed(_):
+    assert 'Respondent:' in browser.html
+    respondent = browser.find_by_id('completed-respondent')[0].value
+    assert respondent == 'first_name last_name'
```

Comment:
Minor point , and not a function of this change ( so no change needed) , but using constant values for first name and last name and then testing that value could give misleading test results under some circumstances